_components/tooltip/guidance/usability.md

```diff
@@ -1,4 +1,3 @@
-- **Use affordances.** A hidden tooltip is unusable. Use tooltips only on elements that appear interactive, like buttons or links.
```

Comment:
@jaclinec Wanted to check - Is it reasonable to remove this line now that we allow tooltip on non-interactive elements?